src/main.rs

```diff
@@ -99,6 +97,18 @@ pub struct Args {
     )]
     pub no_hint: bool,
 
+    #[arg(
+        short = 'c',
+        long = "config-file",
+        help = "Path to the configuration file.",
+        long_help = "
+        Specifies the path to the configuration file.
+        ",
+        default_value = "~/.config/jnv/config.toml",
+        value_parser = |x: &str| Ok::<String, std::convert::Infallible>(shellexpand::tilde(x).into_owned()),
+    )]
+    pub config_file: String,
```

Comment:
I’d like to place an example of the actual configuration file somewhere (in `examples/`?).

Reply:
Good idea